pkg/controller/cassandracluster/pod_operation.go

```diff
@@ -685,9 +685,10 @@ func (rcc *ReconcileCassandraCluster) runRebuild(hostName string, cc *api.Cassan
 
 	if labelSet != true {
 		err = errors.New("operation-argument is needed to get the datacenter name to rebuild from")
-	} else if cc.IsValidDC(rebuildFrom) == false {
-		err = fmt.Errorf("%s is not an existing datacenter", rebuildFrom)
 	}
+	/*} else if cc.IsValidDC(rebuildFrom) == false {
+		err = fmt.Errorf("%s is not an existing datacenter", rebuildFrom)
+	}*/
```

Comment:
@erdrix As the current CassandraCluster does not know anything about datacenter hosted by another remote object, you propose to remove the check. Shouldn't we test it using JMX instead ? Did you try it with a non existing DC and check logs ? If they are explicit I'm 👍, if not we should test it differently

Reply:
It's not really explicit : 

```logs
$ Cannot rebuild from dc5: java.lang.IllegalStateException : Unable to find sufficient sources for streaming range (3396609007184563874,3447869076743942192] in keyspace system_distributed
```